frame/example/src/lib.rs

```diff
@@ -718,7 +718,7 @@ mod tests {
 		type Index = u64;
 		type BlockNumber = u64;
 		type Hash = H256;
-		type Call = ();
+		type Call = Call<Test>;
```

Comment:
Why are all of these suddenly needed? would be waaay better if you implement whatever you need for `()` to allow tests to ignore this.

Reply:
You are right. It is no longer needed with the rewrite.